src/shapes/group.class.js

```diff
@@ -559,27 +559,14 @@
    * @static
    * @memberOf fabric.Group
    * @param {Object} object Object to create a group from
-   * @param {Function} [callback] Callback to invoke when an group instance is created
+   * @returns {Promise<fabric.Group>}
    */
-  fabric.Group.fromObject = function(object, callback) {
+  fabric.Group.fromObject = function(object) {
     var objects = object.objects,
         options = fabric.util.object.clone(object, true);
     delete options.objects;
-    if (typeof objects === 'string') {
-      // it has to be an url or something went wrong.
-      fabric.loadSVGFromURL(objects, function (elements) {
```

Comment:
this feature has to go away.

Reply:
I think this means you can remove `groupSVGElements`